pystiche/papers/ulyanov_et_al_2016/loss.py

```diff
@@ -8,12 +7,22 @@
     MultiLayerEncodingOperator,
 )
 from pystiche.image import extract_batch_size
-from pystiche.loss import MultiOperatorLoss
+from pystiche.loss import PerceptualLoss
 from .utils import ulyanov_et_al_2016_multi_layer_encoder
 
+__all__ = [
+    "UlyanovEtAl2016MSEEncodingOperator",
+    "ulyanov_et_al_2016_content_loss",
+    "UlyanovEtAl2016GramOperator",
+    "ulyanov_et_al_2016_style_loss",
+    "ulyanov_et_al_2016_perceptual_loss",
+]
+
 
 class UlyanovEtAl2016MSEEncodingOperator(MSEEncodingOperator):
-    def __init__(self, encoder: Encoder, score_weight: float = 1e0):
+    def __init__(
+        self, encoder: Encoder, impl_params: bool = True, score_weight: float = 1e0
+    ):
```

Comment:
@jbueltemeier `impl_params` was missing from the constructor. This was just a typo, correct?

Reply:
Superseded by #167.